src/TwigComponent/doc/index.rst

```diff
@@ -1553,6 +1553,12 @@ are called additional times, the cached value is used.
     Computed methods only work for component methods with no required
     arguments.
 
+.. warning::
+
+    Ensure to not use the ``ExposeInTemplate`` attribute on a computed method,
+    otherwise the method will be called twice instead of only once, leading to
```

Comment:
It kinda leads to present it as a bug... but it is not illogical.

The calls are not made at the same time so the "twice" is maybe a bit missleading  
(first time to build the context of the templates, before render, and  one by you calling the cache object).

Reply:
Do you think instead, we should fix this behavior? 

I mean, throw an exception if the user tries to call a computed method with `#[ExposeInTemplate]` ?

Reply:
I am not sure we can. BUT we can maybe populate the computed before the render if method as the attribute. There are some limitations but this could work. 

Lets update doc for now and see after if we change and how wdyt ? 

(but id rather not use warning or any tone that can stress or frighten users 😇)